src/Spout/Writer/Common/Cell.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Box\Spout\Writer\Common;
+
+use Box\Spout\Writer\Common\Helper\CellHelper;
+
+class Cell
+{
+    /**
+     * Numeric cell type (whole numbers, fractional numbers, dates)
+     */
+    const TYPE_NUMERIC = 0;
+
+    /**
+     * String (text) cell type
+     */
+    const TYPE_STRING = 1;
+
+    /**
+     * Formula cell type
+     */
+    const TYPE_FORMULA = 2;
+
+    /**
+     * Blank cell type
+     */
+    const TYPE_BLANK = 3;
```

Comment:
There is now "Blank" and "Empty". If you think "Blank" is a better name, can you rename the isEmpty method from CellHelper to isBlank?
I don't have any preference so feel free to pick whatever you like the best :)